netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -107,6 +109,21 @@ public static SslContextBuilder forServer(File keyCertChainFile, File keyFile) {
   /**
    * Creates a SslContextBuilder with ciphers and APN appropriate for gRPC.
    *
+   * @see SslContextBuilder#forServer(InputStream, InputStream)
+   * @see #configure(SslContextBuilder)
+   */
+  public static SslContextBuilder forServer(InputStream keyCertChain, InputStream key) {
+    try {
+      return configure(SslContextBuilder.forServer(keyCertChain, key));
+    } finally {
+      GrpcUtil.closeQuietly(keyCertChain);
```

Comment:
Normally the creator of the `InputStream` closes (i.e., with try-with-resources). But also this is supposed to act effectively the same as Netty's SslContextBuilder. I don't think we should be closing here.